qiime2/core/archive/tests/test_provenance.py

```diff
@@ -100,6 +100,7 @@ def test_chain_with_merged_artifact_metadata(self):
                                              md_artifact2.uuid)
 
         # Check action files for uuid-metadata values
+        # FAIL: This test currently fails right here
```

Comment:
This test fails, by the looks of it the artifacts on the merged metadata aren't actually being added to provenance, but I'm not too familiar with how this stuff works, and haven't had the time to really dig into why this may be happening

Reply:
The assert right below this one does pass, it's only this one assertion that fails

Reply:
Sounds like provenance recording might not be wired up quite right anymore. Maybe one way for you to debug is to reproduce this data in an REPL, then look at `action/action.yaml` to see how things are being recording w.r.t. metadata.